src/lib/Logger.ts

```diff
@@ -1,18 +1,29 @@
 import * as winston from 'winston';
+import { isMatch } from 'micromatch';
 
 process.stdout.isTTY = true;
 
 const ProdLogger = winston.createLogger({
-  level: 'info',
+  level: 'http',
   format: winston.format.json(),
   transports: [new winston.transports.Console()],
 });
-const DevelopmentLogger = winston.createLogger({
-  level: 'info',
-  format: winston.format.combine(winston.format.colorize(), winston.format.simple()),
+
+const DevLogger = winston.createLogger({
+  level: 'verbose',
```

Comment:
So we'll only get all those extra logs in dev?
Maybe we'll need to move a few key ones out of verbose and into info then?

Just so that we're getting enough info in our prod logs...
At the same time, I dont want to dump too much, not sure what the limits are, but people definitely hit them

Reply:
I'll have a look through the ones I've made `verbose` and see if any could be useful in a prod scenario